### ISSUE TITLE
HAR format

### ISSUE THREAD

**Issue by mderazon** [ChromeDevTools/timeline-viewer#39]
Does this suppose to support the HAR format exported by chrome dev tools in the network tab ?

If it does, I can't get that to work

**Comment by denar90** [comment]
Hi. We don't support this feature right now and I'm not sure we will... cc @paulirish

**Comment by paulirish** [comment]
no, but now that devtools frontend supports importing HAR it will be possible in the future.

**Comment by mderazon** [comment]
Haven't seen that option in devtools (latest Chrome 60)

Also, there are many network timeline analyzer tools available, but IMO the best thing about this tool is that I can share a **direct link** with a co-worker without them needing to import any file or do anything (assuming I upload the file first to Dropbox/Gist etc). I don't think I have seen any other HAR analyzer do that.
